hicexplorer/hicAdjustMatrix.py

```diff
@@ -111,19 +111,17 @@ def adjustMatrix(pArgs):
         # log.debug('genomic_regions {}'.format(genomic_regions))
         matrix_indices_regions = []
         for region in genomic_regions:
-            _regionBinRange = hic_matrix.getRegionBinRange(region[0], region[1], region[2])
+            _regionBinRange = hic_matrix.getRegionBinRange(region[0],
```

Comment:
Please have it in one line and change your linter and autolint settings to the rules we have defined in travis.yaml

Reply:
Please move the changes of hicAdjustMatrix to its own PR and add a test case which shows why this change is necessary.

Reply:
could you please check why the test from chic background model fails?